src/generic/falsy.ts

```diff
@@ -1 +1 @@
-export type Falsy = false | 0 | -0 | 0n | '' | null | undefined;
+export type Falsy = false | 0 | -0 | 0n | '' | null | undefined | 0x0n | 0x0 | -0x0;
```

Comment:
This addition isn't necessary. The `0x` notation is just a different notation, the value is the same, as you can see in this playground https://www.typescriptlang.org/play?#code/GYVwdgxgLglg9mABAQwBQEoBciAMAPHRAbwFgAoRRAJwFMoQqkcBucgX3PNElgUQCMM2QqQrU6DJgVZk2QA